src/internal/connector/exchange/restore_test.go

```diff
@@ -15,13 +15,16 @@ import (
 	"github.com/alcionai/corso/src/internal/connector/mockconnector"
 	"github.com/alcionai/corso/src/internal/connector/support"
 	"github.com/alcionai/corso/src/internal/tester"
+	"github.com/alcionai/corso/src/pkg/account"
 	"github.com/alcionai/corso/src/pkg/control"
 	"github.com/alcionai/corso/src/pkg/path"
 )
 
 type ExchangeRestoreSuite struct {
 	suite.Suite
-	gs graph.Servicer
+	gs          graph.Servicer
```

Comment:
can this be removed?

Reply:
Nope, still currently in use by Restore funcs.  I'm not focused on wholesale replacement of all graph.Service instances at the moment.  The goal of this pr chain is to get us to mocked testing around incrementals.  Those don't involve the restore path, so things will be in a halfway state until we come back for cleanup.